parser/go.sum

```diff
@@ -1,7 +1,11 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
I wonder why go.sum changed.
This PR seems not introducing third-party packages?

Reply:
It might be mis-imported by a commit that directly use github.com/tjfoc/gmsm. I have reverted it back.